browser_history/cli.py

```diff
@@ -23,37 +24,56 @@ def make_parser():
                                             Checkout the GitHub repo https://github.com/pesos/browser-history
                                             if you have any issues or want to help contribute''')
 
+    parser_.add_argument('-t','--type',
+                        default='all',
+                        help=f'''
+                                argument to decide whether to retrieve history or bookmarks.
+                                Should be one of all, {AVAILABLE_TYPES}.
+                                Default is all (gets both histories and bookmarks)
+                                .''')
     parser_.add_argument('-b', '--browser',
                          default='all',
                          help=f'''
-                                browser to retrieve history from. Should be one of all, {AVAILABLE_BROWSERS}.
-                                Default is all (gets history from all browsers).''')
+                                browser to retrieve history or bookmarks from. Should be one of all, {AVAILABLE_BROWSERS}.
+                                Default is all (gets history or bookmarks from all browsers).''')
 
     parser_.add_argument('-f', '--format',
                          default="csv",
                          help=f'''
                                 Format to be used in output. Should be one of {AVAILABLE_FORMATS}.
                                 Default is csv''')
 
-    parser_.add_argument('-o', '--output',
+    parser_.add_argument('-h_o', '--history_output',
```

Comment:
Nothing wrong here but it's not common convention that flags have (Underscore)`_` in them. We could go by using `h` for this but then, we can't use `b` for bookmarks as `b` is already reserved for `browser`. 

I would say it's best if we don't implement the single letter option for now, Lets just go with the `--history_output` and `--bookmarks_output`. Maybe if you / @Samyak2 / anybody reviewing this PR gets a better idea, we can think about it.

Reply:
Okay!

Reply:
How is -bm?